### PR TITLE
test: update lua client to 0.0.1-13

### PR DESCRIPTION
**Pull request by justinmk**

### CODE REVIEW

File: third-party/cmake/BuildLuarocks.cmake [hunk 1]
```diff
@@ -122,7 +122,7 @@ if(USE_BUNDLED_BUSTED)
 
   add_custom_command(OUTPUT ${HOSTDEPS_LIB_DIR}/luarocks/rocks/nvim-client
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build https://raw.githubusercontent.com/neovim/lua-client/0.0.1-12/nvim-client-0.0.1-12.rockspec ${LUAROCKS_BUILDARGS} LIBUV_DIR=${HOSTDEPS_INSTALL_DIR}
+    ARGS build https://raw.githubusercontent.com/neovim/lua-client/0.0.1-13/nvim-client-0.0.1-13.rockspec ${LUAROCKS_BUILDARGS} LIBUV_DIR=${HOSTDEPS_INSTALL_DIR}
```
Comment (justinmk) [inline third-party/cmake/BuildLuarocks.cmake]: @tarruda Is there a reason why the file `nvim-client-0.0.1-13.rockspec` cannot be named simply `nvim-client.rockspec` ?

**Comment by fwalch** [comment]
Tried restarting the build, but:

> CMake Error: The current CMakeCache.txt directory /home/travis/build/neovim/neovim/deps-build/CMakeCache.txt is different than the directory /home/travis/nvim-deps where CMakeCache.txt was created. This may result in binaries being created in the wrong place. If you are not sure, reedit the CMakeCache.txt

@justinmk if you rebase on `master`, it should work.

**Comment by fwalch** [comment]
I see you already noticed that `master´ broke; I think this is because CMake only picks up changes to variables and thus didn't use the new nvim-client. Also locally, I had do do `make distclean` first, then it was picked up. Now I have the same error as current master. It only occured because I deleted the cache on master, therefore it tried to rebuild from scratch.. sorry about that.

**Comment by justinmk** [comment]
No problem. I got confused because `continuous-integration/travis-ci/pr` appeared successful (but it was old, I think). The `continuous-integration/travis-ci/push` check actually failed, but since I'm used to QB build failing all the time I did not see it.

**Comment by fwalch** [comment]
The `push` check is the build Travis did on `master` after you merged this, so you couldn't have known before that this PR would cause an issue.

**Pull request merged**